packages/shared-ui/src/styles/index.css

```diff
@@ -141,6 +145,10 @@
   border: 1px solid var(--cb-primary-color);
 }
 
+.cb-input:disabled {
```

Comment:
Did Adam ask you to adapt the styles of cb-input:disabled, cb-phone-input-field-disabled, cb-phone-input-field-button-disabled, cb-phone-input-field-input:disabled and .cb-container? As the component is used in production now we have to be really careful with these styling changes that affect the component.

Reply:
I talked to Adam. For now, we will keep this change since no other screen is using disabled input fields and maybe in future use different color.